tools/modm_tools/openocd.py

```diff
@@ -78,7 +81,8 @@ def call(commands=None, config=None, search=None, blocking=True, silent=False, v
     # See http://openocd.org/doc/html/Running.html
     # os.environ.get("OPENOCD_SCRIPTS", "")
 
-    command_openocd = "openocd {} {} {}".format(
+    command_openocd = "{} {} {} {}".format(
+        binary,
```

Comment:
```suggestion
        os.environ.get("MODM_OPENOCD_BINARY", "openocd"),
```
But wouldn't this do the same thing much easier and build system independent?